package.json

```
@@ -15,7 +15,7 @@
   "scripts": {
     "build": "bash ./scripts/md2html/build.sh",
     "build-src": "npm run validate-markdown && bash ./scripts/md2html/build.sh src && bash ./scripts/schema-publish.sh src",
-    "test": "c8 --100 vitest --watch=false && bash scripts/schema-test-coverage.sh",
+    "test": "c8 --100 vitest run --coverage && nyc check-coverage",
```

```suggestion
    "test": "c8 --100 vitest run --coverage && ([ \"$BASE\" = \"dev\" ] || nyc check-coverage)",
```
We need to skip `nyc` for pull requests targeting the `dev` branch